docs/rector_rules_overview.md

```diff
@@ -18,8 +18,11 @@ return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
 
     $services->set(AddArgumentDefaultValueRector::class)
-->configure([                new AddArgumentDefaultValue('SomeClass', 'someMethod', 0, false),
-]);
+        ->call('configure', [[
+            AddArgumentDefaultValueRector::ADDED_ARGUMENTS => ValueObjectInliner::inline([
+                new AddArgumentDefaultValue('SomeClass', 'someMethod', 0, false),
+            ]),
+        ]]);
```

Comment:
This is changed when I ran `composer docs` Let me know if you want me to revert this.

Reply:
It requires this update if composer.json

https://github.com/rectorphp/rector-symfony/commit/041b6a55afbf249daea857459a978dd8308dd72d#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34

Reply:
Should that be done in separate PR or in this one?

Reply:
Both options are fine, it's a 1 line fix